src/App.tsx

```diff
@@ -1,156 +1,92 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { getTodos } from './api/todos';
+import { Header } from './components/Header/Header';
+import { TodoList } from './components/TodoList/TodoList';
+import { Footer } from './components/Footer/Footer';
+import { FilterStatus } from './types/FilterStatus';
+import { Error } from './components/ErrorBlock/Error';
 
-const USER_ID = 0;
+const USER_ID = 10266;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState<Error | null>(null);
+  const [filter, setFilter] = useState(FilterStatus.ALL);
+
+  const handleCloseError = () => {
+    setError(null);
+  };
+
+  const filteredTodos = todos.filter((todo) => {
+    switch (filter) {
+      case FilterStatus.ACTIVE:
+        return !todo.completed;
+      case FilterStatus.COMPLETED:
+        return todo.completed;
+      default:
+        return true;
+    }
+  });
+
+  useEffect(() => {
+    async function fetchTodos() {
+      try {
+        setError(null);
+        const response = await getTodos(USER_ID);
+
+        setTodos(response);
+      } catch (err) {
+        setError(err as Error);
+      }
+    }
+
+    fetchTodos();
+  }, []);
+
+  useEffect(() => {
```

Comment:
The timeout for hiding the error notification should be in the 'Error' component itself, rather than in the 'App' component.